ui/applets/kit/src/components/Tabs.tsx

```diff
@@ -14,6 +14,7 @@ export interface TabsProps extends VariantProps<typeof tabsVariants> {
   keys?: string[];
   selectedTab?: string;
   layoutId: string;
+  keyComponent?: { key: string; component: React.ReactNode }[];
```

Comment:
There is not need of keyComponent neither customComponent since Tabs accept an array of tabs and you could use any custom component as children.